Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/AssetPage.tsx

```diff
@@ -253,19 +253,19 @@ export default function AssetPage(props: IProps) {
 
     function showAttributes(): JSX.Element {
         if (newEditAsset.AssetType == 'Breaker')
-            return <BreakerAttributes NewEdit={newEdit} Asset={newEditAsset as OpenXDA.Types.Breaker} UpdateState={setNewEditAsset} />;
+            return <BreakerAttributes NewEdit={newEdit} Asset={newEditAsset as OpenXDA.Types.Breaker} UpdateState={setNewEditAsset} Disabled={newEditAsset?.ID === 0}/>;
```

Comment:
Apologies if I misunderstood something, but it seems that your commit is titled
"Disabled Asset Properties for new Assets in NMW"
but the ticket is titled
"Make editing existing asset fields in NMW impossible"
These seem like opposite wants. Has something changed since the ticket was made?

Reply:
Yea error in the commit title. Should be disable for existing Assets in NMW

Reply:
If this is the case, then shouldn't this be something like this?
```suggestion
            return <BreakerAttributes NewEdit={newEdit} Asset={newEditAsset as OpenXDA.Types.Breaker} UpdateState={setNewEditAsset} Disabled={newEditAsset?.ID > 0}/>;
```
since if it exists already this should be disabled?

Reply:
Yes